pkg/activator/activator.go

```diff
@@ -18,11 +18,13 @@ package activator
 
 const (
 	// K8sServiceName is the name of the activator service
-	K8sServiceName          = "activator-service"
+	K8sServiceName = "activator-service"
+	// RequestCountHTTPHeader is the header key for number of tries
```

Comment:
Does it make sense to document the reason for having two distinct headers here?

Reply:
I added a note suggesting that the ResponseCountHTTPHeader should be removed in the future.